setup.py

```diff
@@ -18,10 +18,14 @@
               "centinel.vpn"],
     install_requires=["argparse >= 1.2.1",
                       "dnspython >= 1.11.0",
-                      "requests >= 2.5.1",
+                      "requests >= 2.9.1",
                       "trparse >= 0.2.1",
                       "pycurl >= 7.19.5",
-                      "urllib3 == 1.9.1"],
+                      "urllib3 == 1.9.1",
+                      "dnspython == 1.12.0",
+                      "M2Crypto == 0.21.1",
```

Comment:
Remove this. This one is optional (TLS will be disabled if this isn't installed).